package.json

```diff
@@ -7,7 +7,9 @@
 	"bin": "dist/cli.js",
 	"scripts": {
 		"prepare": "npm t",
-		"build": "microbundle --target node -f cjs --no-compress src/index.js src/cli.js src/appender.js",
+		"build": "npm run build:node && npm run build:web",
+		"build:node": "microbundle --target node -f cjs --no-compress src/index.js src/cli.js src/appender.js",
+		"build:web": "microbundle -f iife --no-compress --external none --alias jest-message-util=C:\\code\\github\\developit\\karmatic\\src\\lib\\jest\\messageUtilFake.js --define process.env.NODE_ENV=production -i src/lib/jest-globals.js -o dist/lib/jest-globals.js",
```

Comment:
I'm bundling `jest-globals.js` separately at build time to avoid our users having to re-bundle all the jest stuff on each run of karmatic. Probably need to move some of these deps into `devDeps` since they are bundled. Or maybe move them to `bundledDeps`?

Also need to figure out/fix how to specify aliases in microbundle so that it doesn't require absolute paths